### PR TITLE
set laser topic from launch file

### PR DESCRIPTION
**Pull request by mehditlili**

### CODE REVIEW

**Comment by mehditlili** [comment]
@jihoonl I created a new recovery behavior that allows the robot to move back when stuck, Is turtlebot_apps the right folder?

**Comment by jihoonl** [comment]
Nice work @mehditlili. This recovery_behavior is usable in general I think. How about making PR to [upstream navigation stack](https://github.com/ros-planning/navigation)?  @stonier how do you think?

File: stepback_recovery/include/stepback_recovery/stepback_recovery.h [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+#include <ros/ros.h>
+#include <base_local_planner/costmap_model.h>
+#include <geometry_msgs/Twist.h>
+#include <geometry_msgs/Point.h>
+#include <geometry_msgs/TransformStamped.h>
+#include <angles/angles.h>
+
+namespace stepback_recovery{
+  /**
+   * @class RotateRecovery
```
Comment (jihoonl) [inline stepback_recovery/include/stepback_recovery/stepback_recovery.h]: The code documentation refers wrong information. Could you update this?

File: stepback_recovery/package.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<package>
+    <name>stepback_recovery</name>
+    <version>1.11.13</version>
+    <description>
+
+        This package provides a recovery behavior for the navigation stack that attempts to clear space by performing a 360 degree rotation of the robot.
+
+    </description>
+    <author>Eitan Marder-Eppstein</author>
```
Comment (jihoonl) [inline stepback_recovery/package.xml]: Maintainter and author should be your name. Also description is wrong.

**Comment by jihoonl** [comment]
I making two PR make sense. One for adding laser scan topic. and the other for adding stepback recovery.  

Could you make new PR to add laser scan configuaration? and discuss how shall we add stepback recovery. 

And note the `stepback_recovery` code includes out-dated informations. Could you update them as well?

**Comment by mehditlili** [comment]
I removed my commits for stepback_recovery so you can merge this pr as adding laser scan topic pr. Then I will do a second pull request.

**Comment by jihoonl** [comment]
:+1:

**Pull request merged**